xml/admin_integration.xml

```diff
@@ -30,14 +30,14 @@
   <itemizedlist>
    <listitem>
     <para>
-     &productname; can access the SES cluster.
+     The &productname; cluster can access the SES cluster.
     </para>
    </listitem>
    <listitem>
     <para>
-     &productname; can communicate with the SES nodes&mdash;master, monitoring
-     nodes, OSD nodes and the meta data server in case you need a shared
-     file system. For more details regarding SES refer to
+     That &productname; can communicate with the SES nodes&mdash;master,
```

Comment:
"That" seems wrong here.

Reply:
No, it was intentional, but I have revised it to be consistent with the previous line.